### ISSUE TITLE
Mass downloading for favicons

### ISSUE THREAD

**Issue by vit9696** [keepassium/KeePassium#320]
After updating to a new version of KeePassium it is now possible to download favicons. Thank you so much for this. I would personally like to get favicons for websites without them. For this I suggest the following changes:

- [ ] Add context menu entry on the item to download a favicon.
- [ ] Enable mass item selection in the item list.
- [ ] Introduce advanced search for items without favicons.

**Comment by keepassium** [comment]
Makes sense. I would not want to clutter the context menu for each item, but mass selection + a dedicated menu for selected items should do the trick.

**Comment by keepassium** [comment]
Downloading favicons for selected entries is done in v1.54.155 via b9ee48bc3e7c78ee0ac9a1a30eacd262b9bd6800.
